### ISSUE TITLE
sqlite not being written..

### ISSUE THREAD

**Issue by jult** [mghaan/mikrotik-traffic#1]
I get this in nginx access log:
`
192.168.1.1 - - [25/Aug/2023:01:27:57 +0200] "GET /mtstore.pl?id=core_rtr&tx=2 969 274 818&rx=42 035 730 259 HTTP/1.1" 400 150 "-" "-"
`
The router apparently is sending its data correctly, but the data is not added to the sqlite file under /var/www/cgi-bin/

SQLite3 is working on the server
~# sqlite3
SQLite version 3.40.1 2022-12-28 14:03:47
Enter ".help" for usage hints.
Connected to a transient in-memory database.
Use ".open FILENAME" to reopen on a persistent database.

Any idea?

**Comment by mghaan** [comment]
The only reason I can think of is that mtstore.pl can't write into sqlite file. I'm afraid I won't be much helpful as I no longer use this method and instead I switched to method using SNMP and InfluxDB as storage/graphing. Once setup, it's much easier to add/remove devices. I'll update this project soon describing the new solution.

**Comment by jult** [comment]
Yes, I too just switched to using librenms and snmp. It's better, because I can monitor more than just this router's traffic ;) Thanks anyway.
